sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/FieldDeclarationsCountCheckNOK.java

```diff
@@ -0,0 +1,12 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FieldDeclarationsCountCheckNOK {
+
+    // both FQN and import'ed type names work:
+    Logger logger1;
```

Comment:
please make a test case with "Logger" from different(not slf4j) package in imports.
there should be no violation.
Check have to process imports to know what short name is actually used, we have examples in other Checks for this.